client/src/test/java/tds/student/StudentTest.java

```diff
@@ -1,18 +0,0 @@
-package tds.student;
-
-import org.junit.Test;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-public class StudentTest {
-
-    @Test
-    public void shouldBeCreated() {
```

Comment:
I removed it a bit out of laziness during the refactor as I was modifying it quite a bit.  I've added the test back in verifying the builder works as expected.